dist/index.html

```diff
@@ -26,28 +26,27 @@ <h3>or Search by keyword</h3>
         </ul>
       </section>
     </section>
-    <section class="results">
+    <section class="results hidden">
```

Comment:
Could you switch it so that this starts out showing by default, and the other section starts out hidden?